readme.md

```
@@ -1,620 +1,276 @@
 
----
+Please report security vulnerabilities privately using [GitHub Security Advisories](https://github.com/dgtlss/warden/security/advisories/new).
 
-<div align="center">
+## License
 
-**Made with ❤️ for the Laravel community**
+Warden is open-sourced software licensed under the [MIT license](LICENSE).
 
-[⭐ Star on GitHub](https://github.com/dgtlss/warden) | [📦 Packagist](https://packagist.org/packages/dgtlss/warden) | [🐦 Follow Updates](https://twitter.com/nlangerdev)
+## License
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)